app/lib/placeHelper.js

```diff
@@ -2,11 +2,11 @@ const log = require('./logger');
 
 const localOrder = {
   City: 5,
-  Town: 4,
-  Village: 3,
   Hamlet: 2,
   'Other Settlement': 1,
   'Suburban Area': 0,
+  Town: 4,
+  Village: 3,
```

Comment:
The alphabetisation has made the order of importance harder to read

Reply:
Good spot. Changes made.